### PR TITLE
fix(sql): Add column

### PR DESCRIPTION
**Pull request by RockySouthpaw**
**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [ ] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ ] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by RockySouthpaw** [comment]
Probably needs to be adjusted to check for the current table or we could remove this querry all together and handle this within starting the phone so the column exist based off the configured database stuff.

File: import.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # comment this out if your server has a different table/location for phone_number
 # or if it already exists in the users table
-# ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) DEFAULT NULL;
+ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) DEFAULT NULL;
```
Comment (itschip) [inline import.sql:3]: We have to check if the table exists.

**Review by itschip — CHANGES_REQUESTED** [review]

File: import.sql (hunk 1 above)
Comment (RockySouthpaw) [reply import.sql:3]: > We have to check if the table exists.

Isnt those checks dependent on MYSQL versions?

Comment (itschip) [reply import.sql:3]: I don't think 'if exists' is

Comment (itschip) [reply import.sql:3]: I don't think 'if exists' is

**Comment by TasoOneAsia** [comment]
Something for me to tackle during stream today

**Comment by RockySouthpaw** [comment]
> Something for me to tackle during stream today

And did you?

**Comment by itschip** [comment]
> > Something for me to tackle during stream today
> 
> And did you?

smh

**Comment by itschip** [comment]
@RockySouthpaw Just run a 'if exsits' and call it a day

**Comment by RockySouthpaw** [comment]
effort

**Pull request closed without merging**